cgroup/cgroup.go

```diff
@@ -24,32 +26,64 @@ type cgroup struct {
 }
 
 func (d cgroup) AddDevicesAllowed(containerID string, permission string) (bool, *dbus.Error) {
-	// Make sure path is relative to cgroupFSDockerDevices
-	allowedFile, err := securejoin.SecureJoin(cgroupFSDockerDevices, containerID+string(filepath.Separator)+"devices.allow")
-	if err != nil {
-		return false, dbus.MakeFailedError(fmt.Errorf("Security issues with '%s': %s", containerID, err))
-	}
+	// Check for CGroups v2
+	if _, err := os.Stat("/sys/fs/cgroup/cgroup.controllers"); err == nil {
```

Comment:
Maybe make sense to handle it on InitializeDBus (I guess once we can rename that to setup or so) and set a variable at the scope of this file?